addons/apollo/functions/fnc_getPlayerInfo.sqf

```diff
@@ -61,6 +63,7 @@ if (_loadData == "loaded") then {
 if !(_success) exitWith {
     ERROR_2("Failed to load info (Name: %1 - UID: %2)!",profileName,getPlayerUID _player);
     ["Could not load info"] call CBA_fnc_notify;
```

Comment:
Should probably not be in this function, but where this function is called, so you can personalize the message.